pyvista/core/filters/data_set.py

```diff
@@ -3872,14 +3866,14 @@ def sample_over_multiple_lines(self, points, tolerance=None, progress_bar=False)
 
         Parameters
         ----------
-        points : np.ndarray or list
+        points : sequence[float]
```

Comment:
According to the example this can be a list of lists. Perhaps it should be another `array_like`?
```suggestion
        points : array_like[float]
```

Reply:
Yes, this is a good candidate for `array_like`.